druid/src/widget/widget_ext.rs

```diff
@@ -273,6 +273,18 @@ pub trait WidgetExt<T: Data>: Widget<T> + Sized + 'static {
     fn scroll(self) -> Scroll<T, Self> {
         Scroll::new(self)
     }
+
+    /// Wrap this widget in a [`DisabledIf`] widget.
+    /// The provided closure will determine if the widget is disabled.
```

Comment:
We want a newline separating the first 'summary' line and the rest of the text; the first line is displayed beside the item if it is included in a list in the docs.

```suggestion
    /// Wrap this widget in a [`DisabledIf`] widget.
    ///
    /// The provided closure will determine if the widget is disabled.
```